applications/API/models.py

```
@@ -0,0 +1,625 @@
+
+    Args:
+        number (int): The positive integer to encode.
+
+    Returns:
+        str: The base64-encoded string representation of the input number.
+
+    Raises:
+        Base64EncoderException: If the input number is not positive.
+    """
+    if number <= 0:
+        raise Base64EncoderException("Number must be non-negative")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)